src/app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\PostRequest;
+use App\Models\Post;
+
+class PostController extends Controller
+{
+
+    public function create(PostRequest $request)
+    {
+        $post = new Post();
+        $post->content = $request->input('content');
+        $post->user_id = auth()->id();
+
+        if($request->hasFile('image')) {
```

Comment:
Follow PSR

Reply:
I have resolved this comment, sir. Thank you! 🙇